.github/workflows/update-pinned-reqs.yml

```diff
@@ -24,11 +24,17 @@ jobs:
         python-version: 3.12
 
     - name: Install tox and its extensions
-      run: python -m pip install --upgrade tox tox-uv
+      run: python -m pip install --upgrade tox tox-uv nox uv
 
-    - name: Rebuild requirements
+    # Temporarily use requirements files generated both by tox and nox
+    # during the switchover from tox to nox.
```

Comment:
Because the requirements filenames are changing, we'll need the files generated by tox for the remaining tox environments, as well as the files generated by nox for the current and forthcoming nox sessions.  It'll be messier having duplicate requirements files under different names, but I want to switch things over completely soon.